pyfstat/mcmc_based_searches.py

```diff
@@ -284,28 +284,17 @@ def logp(self, theta_vals, theta_prior, theta_keys, search):
         return np.sum(H)
 
     def logl(self, theta, search):
+        in_theta = copy.copy(self.fixed_theta)
         for j, theta_i in enumerate(self.theta_idxs):
-            self.fixed_theta[theta_i] = theta[j]
-        twoF = search.get_fullycoherent_twoF(
-            self.minStartTime, self.maxStartTime, *self.fixed_theta
-        )
+            in_theta[theta_i] = theta[j]
+        twoF = search.get_fullycoherent_twoF(*in_theta)
```

Comment:
I feel quite uncomfortable having this call without keywords; but you said that would be a future rework, so fine by now.

Reply:
Yeah, at least it's no worse than before.